src/__experimental__/components/textbox/textbox.component.js

```diff
@@ -19,7 +19,7 @@ const Textbox = ({
   ...props
 }) => {
   return (
-    <FormField { ...props }>
+    <FormField value={ value } { ...props }>
```

Comment:
does that component need a value? I can't find where it's used.

Reply:
Yeah it's a hangover from something I was trying before, I'll remove it